src/vm/arm64/crthelpers.asm

```diff
@@ -87,68 +134,89 @@
 ; as C++ method.
 
     LEAF_ENTRY JIT_MemSet
-    uxtb        w8,w1
-    sxtw        x8,w8
-    orr         x8,x8,x8 lsl #8
-    orr         x8,x8,x8 lsl #0x10
-    orr         x9,x8,x8 lsl #0x20
-    and         x8,x0,#7
-    cbz         x8,JIT_MemSet_0x7c
-    and         x8,x0,#3
-    cbnz        x8,JIT_MemSet_0x38
-    str         w9,[x0]
-    add         x0,x0,#4
-    mov         x8,#-4
-    add         x2,x2,x8
-    b           JIT_MemSet_0x7c
-JIT_MemSet_0x38
-    cbz         x2,JIT_MemSet_0x7c
-    tbnz        x0,#0,JIT_MemSet_0x60
-JIT_MemSet_0x40
-    and         x8,x0,#7
-    cbz         x8,JIT_MemSet_0x7c
-    strh        w9,[x0]
-    add         x0,x0,#2
-    mov         x8,#-2
-    add         x2,x2,x8
-    cbnz        x2,JIT_MemSet_0x40
-    b           JIT_MemSet_0x7c
-JIT_MemSet_0x60
-    and         x8,x0,#7
-    cbz         x8,JIT_MemSet_0x7c
-    strb        w9,[x0]
-    add         x0,x0,#1
-    mov         x8,#-1
-    add         x2,x2,x8
-    cbnz        x2,JIT_MemSet_0x60
-JIT_MemSet_0x7c
-    cmp         x2,#8
-    blo         JIT_MemSet_0xb8
-    lsr         x8,x2,#3
-    mov         x11,x8
-    mov         x10,x0
-    add         x8,x10,x11 lsl #3
+    ands        w8, w1, #0xff
+    mrs         x3, DCZID_EL0                      ; x3 = DCZID_EL0
+    mov         x6, #4
+    lsr         x11, x2, #3                        ; x11 = count >> 3
+
+    orr         w8, w8, w8, lsl #8
+    and         x5, x3, #0xf                       ; x5 = dczid_el0.bs
+    csel        x11, x11, xzr, eq                  ; x11 = (val == 0) ? count >> 3 : 0
```

Comment:
Yes the assembler is complaining about the `csel`
```
C:\DATA\clr\try_new_toolchain\coreclr\bin\obj\Windows_NT.arm64.Checked\src\vm\wks\CrtHelpers.asm(3498) : error A2506: Unknown error
    csel        x11, x11, xzr, eq                  ; x11 = (val == 0) ? count >> 3 : 0
```

Should I inject the raw encoding? How?